src/main/java/team4/footwithme/chat/service/ChatMemberServiceImpl.java

```diff
@@ -152,9 +191,27 @@ public ChatMemberResponse leaveChatMember(ChatMemberServiceRequest request) {
     @Override
     @Transactional
     public String leaveChatRoom(Long chatroomId) {
-        Chatroom chatroom = getChatroom(chatroomId);
+        Chatroom chatroom = getChatroomByChatroomId(chatroomId);
 
-        chatMemberRepository.deleteByChatRoom(chatroom);
+        chatMemberRepository.updateIsDeletedForChatRoom(chatroom);
+        return "Successfully leaving the chatroom";
+    }
+
+    @Override
+    @Transactional
+    public String leaveTeamChatRoom(Long teamId) {
+        Chatroom chatroom = getChatroomByTeamId(teamId);
+
+        chatMemberRepository.updateIsDeletedForChatRoom(chatroom);
+        return "Successfully leaving the chatroom";
```

Comment:
return 에 사용되는 String들 Enum을 쓴다거나, 상수로 관리하면 좋을 것 같네요 !

`private static final String LEAVE_ROOM_MESSAGE = "Successfully leaving the chatroom";` 이렇게 해두시고 나서

```suggestion
        return LEAVE_ROOM_MESSAGE;
```

이렇게 하는게 더 좋을 것 같아요~ 나중에 재사용성도 그렇고, 날 문자열이 메서드안에 있는것 보다는 낫지않나 생각합니다 !

Reply:
넵! 좋은 의견 감사합니다!